product/strategy.md

```diff
@@ -0,0 +1,108 @@
+---
+navTitle: Strategy
+---
+
+# Product Strategy
+
+Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowForge’s vision. It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowForge is a 100 percent agile company, so our product is; timing is everything. Have a look at our roadmap for what we're pursuing in the upcoming releases. 
+
+### FlowForge for enterprise
+
+The cornerstone of FlowForge is to make Node-RED more accessible to enterprises. Therefore, we are constantly implementing requirements that support the easy use of Node-RED in productive environments. We will focus on:
+
+1. High availability & Scalability
+
+2. State-of-the-art built-in security e.g. VPN support
+
+3. DevOps
+   
+   1. Change control
+   
+   2. Collaborative Development
+   
+   3. Pipelines (e.g. staging)
+
+### Node-RED at edge
+
+Many organizations position Node-RED instances on remote servers, edge or industrial devices, to meet network, infrastructure, or use case requirements. Therefore, management of remote instances is crucial for the overall success of closing the gap between IT and OT. 
+
+FlowForge already supports this process by creating snapshots on Node-RED instances that can be deployed to multiple remote targets. We will continue to expand the functionalities to allow users to manage existing Node-RED instances, improve their development practices to accelerate development and ensure a reliable and repeatable process, providing more flexible remote deployment options and offering the best support for devices in closed and segmented networks as we are facing it in productive environments.
+
+### Certified nodes marketplace
+
+The "Node-RED Library" offers a unique added value to many Node-RED and FlowForge users. It is already possible today to easily create technical implementations out of the box for several OT and IT protocols, thanks to the community and Node-RED library. 
+
+The logical consequence of FlowForge’s journey is to also include “custom nodes” as one element of the product. Therefore, FlowForge will offer official support for certain popular nodes and in addition enable the community to offer commercial “nodes” via FlowForge. Nevertheless, an open-source core is always required and wanted. However, premium extensions and commercial support allow the community to participate in the success of FlowForge.
+
+### IIoT use case library
```

Comment:
I would call this IIoT Solution. Use case library makes these sound like they are documents.

Reply:
Changed with https://github.com/flowforge/handbook/pull/213/commits/7ddc4000e7fae5a59fdf08161ad41b0df4b57f30